sdp-utils.c

```diff
@@ -507,12 +507,12 @@ janus_sdp *janus_sdp_parse(const char *sdp, char *error, size_t errlen) {
 						break;
 					}
 					case 'b': {
-						if(mline->b_name) {
-							if(error)
-								g_snprintf(error, errlen, "Multiple m-line b= lines: %s", line);
-							success = FALSE;
-							break;
-						}
+						//if(mline->b_name) {
+							//if(error)
+							//	g_snprintf(error, errlen, "Multiple m-line b= lines: %s", line);
+							//success = FALSE;
+							//break;
+						//}
```

Comment:
Sorry, but this is not a proper fix either... I can try looking into this, but definitely not before a couple of weeks.